tools/crdb-physical-replication.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+
+# Intended to automate https://docs.google.com/document/d/1vTX6N72J0yL2dDVdyJg063gC761QdZcMQ3ADthFYi8g/edit
+
+bold=$(tput bold)
```

Comment:
`tput` is not sufficiently portable.
Try simply:
```sh
bold=$(printf "\033[1m")
normal=$(printf "\033[m")
```

Reply:
Done